README.md

```diff
@@ -1,11 +1,92 @@
 Rancher Webhook
 ========
- Rancher webhook is both a validating admission webhook and mutating admission webhook for Kubernetes. 
-
+ Rancher webhook is both a validating admission webhook and mutating admission webhook for Kubernetes.
 
 [Explanation of Webhooks in Kubernetes](
 https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
 
+## Background
+The Rancher Webhook is an instance of a Kubernetes admission controller.
+Admission controllers are a standard Kubernetes mechanism to intercept requests to a cluster's
+API server and perform validation or mutation of resources prior to their persistence to
+the cluster database.
```

Comment:
```suggestion
API server and perform validation or mutation of resources prior to their persistence in
the cluster database.
```

Reply:
I think we do say "persist **to** some storage", but I am OK to change this too.

Reply:
Prepositions are complex because there are no clear rules on when to use one over the other. I believe `in` is correct for this usage because we want to specify where resources are stored. `to` is used more often to identify. movement/direction.